src/components/Note/Note.js

```diff
@@ -65,13 +70,12 @@ const Note = ({ annotation }) => {
   return (
     <div ref={containerRef} className={noteClass} onMouseDown={handleNoteClick}>
       <NoteContent annotation={annotation} />
-
-      <div className={repliesClass}>
+      {!isReplyDisabled && <div className={repliesClass}>
```

Comment:
Hmm, I'm not 100% sure(need to check with Matt later) but I think we still want to show existing replies. We just need to disable the textarea that's used to add a reply. Sorry for not making myself clear.

Do you think it'd be better to handle this logic in the `ReplyArea` component(just at the bottom of this file)? If you want to change that component, I think you need to merge dev because Andrey did some change to the component as well.